src/components/SQForm/useForm.tsx

```diff
@@ -128,24 +135,26 @@ export function useForm<TValue, TChangeEvent>({
     if (isFieldError) {
       return (
         <>
-          <WarningIcon color="error" style={SPACE_STYLE} />
-          {errorMessage}
+          <WarningIcon color="error" sx={ICON_STYLES} />
+          <Typography sx={(theme: Theme) => theme.typography.helper}>
```

Comment:
We weren't originally using the theme or a Typography component for these helper text messages. So added that here and made sure we were using the correct styling